pkg/controller/defaults/placements.go

```diff
@@ -213,5 +213,41 @@ var (
 				},
 			},
 		},
+
+		"rgw-multi-instances": rook.Placement{
+			PodAntiAffinity: &corev1.PodAntiAffinity{
+				RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
+					corev1.PodAffinityTerm{
+						LabelSelector: &metav1.LabelSelector{
+							MatchExpressions: []metav1.LabelSelectorRequirement{
+								metav1.LabelSelectorRequirement{
+									Key:      "app",
+									Operator: metav1.LabelSelectorOpIn,
+									Values:   []string{"rook-ceph-rgw"},
+								},
+							},
+						},
+						TopologyKey: "kubernetes.io/hostname",
```

Comment:
we can make use of label which are already defined in kube package https://github.com/kubernetes/api/blob/master/core/v1/well_known_labels.go#L20

Reply:
Thanks @Madhu-1 ... Changed accordingly...